buildSrc/src/main/kotlin/Constants.kt

```diff
@@ -5,9 +5,9 @@ object Constants {
     const val releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
     const val snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
 
-    const val kotlin = "1.6.0"
-    const val dokka = "1.6.0"
-    const val kord = "0.8.0-M8"
+    const val kotlin = "1.6.10"
+    const val dokka = "1.5.31"
```

Comment:
Downgrade for dokka?

Reply:
Must've somehow missed that in the merge.  Has been fixed.